.github/workflows/ci.yml

```diff
@@ -1,66 +1,35 @@
 name: build
 on: [push, pull_request]
 jobs:
-
-  Linux:
-    name: Linux
-    runs-on: ubuntu-latest
+  ci:
+    strategy:
+      fail-fast: true
```

Comment:
This is something I've taken the liberty to add. This will mean that if any of the N tests you're running fails fast it probably means the others will fail too so there's probably no reason why running them all (this should save computation time and therefore $$ to GOA)